service/src/test/java/org/mskcc/smile/service/CorrectCmoPatientIdHandlerTest.java

```diff
@@ -41,7 +41,7 @@ public class CorrectCmoPatientIdHandlerTest {
     private SmilePatientService patientService;
 
     @Container
-    private static final Neo4jContainer databaseServer = new Neo4jContainer<>()
+    private static final Neo4jContainer databaseServer = new Neo4jContainer<>("neo4j:4.4")
```

Comment:
Is this something we can define in the `pom.xml` instead?

Reply:
I think we might be able to. Let me look into this.

Reply:
It looks like we can accomplish this with `application.properties` instead. Let me know if you think otherwise.

Reply:
@ao508 I thought of naming this variable `neo4j_docker_image_version`, but I noticed that it would differ from other `application.properties` variables' dot-separated/snake-case naming convention (e.g. `smile.publishing_failures_filepath`). 

Would `neo4j_docker_image_version` work for you? 

I assume that we're only using the period to avoid conflicts between variables of the same name, but I wasn't sure if there were other reasons.

Reply:
As discussed, I'll try updating `TestContainer` first to see if that would update the default Neo4j image version and let us avoid manually setting it in `application.properties`.